extra/checkout_patch_build.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+SIDE=${SIDE:-backend}
+
+if [[ ! -e openshift-console ]]; then
+  git clone https://github.com/openshift/console.git openshift-console
+else
+  echo 'Using the existing "openshift-console" directory'
+fi
+cd openshift-console
+git checkout ${console_branch}
+
+if [[ -e ../patches/${console_branch} ]]; then
```

Comment:
1. IMHO patches should be optional (manual step or switch in the script).  They may conflict with local changes (assuming you use local console repo) and may be hard to maintain. 
2. we could have one example patch for all versions (there seems  to be no differences)

Reply:
> 1. IMHO patches should be optional (manual step or switch in the script).  They may conflict with local changes (assuming you use local console repo) and may be hard to maintain.
> 
Well at least 1 patch is required - and that is the one to change the build from production to dev (or `build` to `dev-once`).  I took inspiration for it from some ovirt ci scripting. :wink:

Hard to maintain may be true, but the build is setup to apply each patch and fail if one can't be applied.  Happened to me once so far.  It's all a bit of a hack anyway, so I don't mind keeping it a hack.

>
>  2. we could have one example patch for all versions (there seems  to be no differences)
>
Most of the patches are just symlinks.